package.json

```diff
@@ -48,13 +49,14 @@
     "semver": "^4.3.3"
   },
   "devDependencies": {
+    "async": "^0.9.0",
+    "coveralls": "^2.11.2",
+    "gm": "^1.17.0",
+    "istanbul": "^0.3.13",
     "mocha": "^2.2.4",
     "mocha-jshint": "^2.2.0",
-    "istanbul": "^0.3.13",
-    "coveralls": "^2.11.2",
     "node-cpplint": "^0.4.0",
-    "rimraf": "^2.3.2",
-    "async": "^0.9.0"
+    "rimraf": "^2.3.2"
```

Comment:
Alphabetized `devDependencies` for easier scanability and added `gm` to use `gm.compare` in automated tests.